src/Pokemon.js

```diff
@@ -165,12 +165,13 @@ class Pokemon {
   /**
    * [setFavorite description]
    */
-  setFavorite() {
+  setFavorite(status) {
+    status = ((status !== true) && (status !== false)) ? true : status;

```

Comment:
setFavorite(status = true)
instead of 
 status = ((status !== true) && (status !== false)) ? true : status;